hooks/pre_gen_project.py

```diff
@@ -32,7 +32,7 @@
 has_old_zcml = bool([value for value in load_zcml.values() if value])
 if has_old_zcml:
     upgrade_warnings.append(
-        "The 'zcml' dict setting is removed in 2.0, use 'zcml_' prefix variables instead!"
+        "The 'load_zcml' dict setting is deprecated in 2.0, use 'zcml_' prefix variables instead!"
```

Comment:
I think this is more explicit and helpful.

```suggestion
        "The 'load_zcml' dict setting was deprecated in version 2.0. Instead use the name of its keys prefixed with 'zcml_' as the new variable names."
```

Also I don't know if we should update the change log for 2.0.0 to mention that the initial removal was restored in 2.0.1, or clarify the entry for 2.0.1. I missed that on my first reading, and it wasn't really clear at first.

Maybe this?

```
- Fix: Implement a fallback to use ``load_zcml`` dict configuration, effectively making the breaking change of its removal in 2.0.0 a deprecation instead. Fixes #18.
  [@ericof, 2024-03-01]
```

Reply:
I have copied your other suggestion to ``CHANGES.rst`.